packages/ember-glimmer/lib/component.js

```diff
@@ -109,13 +109,384 @@ export const BOUNDS = symbol('BOUNDS');
   elements. This is consistent with the same
   requirement in web components.
 
+    ## HTML Tag
+  The default HTML tag name used for a components's DOM representation is `div`. This
+  can be customized by setting the `tagName` property. The following component
+  class:
+  ```javascript
```

Comment:
The documentation seems to be missing newlines in general, but around code samples specifically.